plugins/confirm.cpp

```diff
@@ -565,7 +580,8 @@ DFhackCExport command_result plugin_enable (color_ostream &out, bool enable)
     }
     if (is_enabled)
     {
-        conf_lua::simple_call("check");
+        //conf_lua::simple_call("check"); this doesn't work anymore, why?
```

Comment:
What do you mean by "doesn't work"? Was it giving an error? It will throw an error deliberately if the list of confirmations between Lua and C++ doesn't line up.

Reply:
Ah good to know. I was getting an error "attempted to call a nil value. stack trace:" (no stack trace) when I enabled the plugin, after I had made a bunch of changes. I eventually traced it to this line and noticed that the rest of the script seemed to work when I commented it out. So I wrote a note to myself to find out why.